### ISSUE TITLE
Updates & new Software on the 3D Printing PC

### ISSUE THREAD

**Issue by NitramLegov** [comakingspace/do-something#274]
The 3D Printing PC is currently not able to to install updates (and thus also new software) using the package manager.

This seems to be due to some signature errors in manjaro-keyring. Maybe someone with greater Linux Skills than myself can take a look :)

Once it works again, we should install the Nextcloud Client to allow file transfers to the 3D Printer PC :-)

**Comment by reogaro** [comment]
I have taken care of the keyring issue and ran a full system update.

I'm not very experienced with Nextcloud though. Maybe someone with greater Nextcloud Skills than myself can take a look @NitramLegov  :^)

**Comment by NitramLegov** [comment]
Thank you! I will take a look into the installation and setup of the nextcloud client :-)

**Comment by NitramLegov** [comment]
I did install nextcloud and set it up to synchronize the drop folder. A shortcut to the folder is created on the desktop.

@reogaro I had to set an empty password for the keyring in order to prevent the system from asking for the keyring password at each startup.
I know it's not ideal, but the alternative would have been to note the keyring password on the screen, which is not significantly more secure 😅
Plus, there is nothing critical on this machine.
